ts/input/tex/base/BaseMethods.ts

```diff
@@ -1641,8 +1641,17 @@ BaseMethods.IndentAlign = function (parser: TexParser, begin: StackItem) {
  * @param {TexParser} parser The calling parser.
  * @param {StackItem} begin The opening stackitem.
  * @param {boolean} numbered True if environment is numbered.
- */
-BaseMethods.Equation = function (parser: TexParser, begin: StackItem, numbered: boolean) {
+ * @param {boolean=} display True if environment is a block display.
+ */
+BaseMethods.Equation = function (
+  parser: TexParser,
+  begin: StackItem,
+  numbered: boolean,
+  display: boolean = true
+) {
+  ParseUtil.checkEqnEnv(parser);
+  parser.configuration.mathItem.display = display;
+  parser.stack.env.display = display;
   parser.Push(begin);
   ParseUtil.checkEqnEnv(parser);
```

Comment:
This is now already checked.

Reply:
Sorry about that.  There was a merge conflict when I cherry-picked this change into a separate branch, and I missed that in resolving the conflict.  I'll remove the second check.

Reply:
Actually, it's the first check that should be removed.